cli/args/flags.rs

```diff
@@ -1734,6 +1735,7 @@ fn compile_args(app: Command) -> Command {
   app
     .arg(import_map_arg())
     .arg(no_remote_arg())
+    .arg(no_npm_arg())
```

Comment:
Can we just have this under `--no-remote` with perhaps `--no-remote` accepting something like `--no-remote=npm`?

Reply:
Actually, we should add a test for this to ensure it doesn’t download with `--no-remote`, but still allows it to work if already cached.

Reply:
> Actually, we should add a test for this to ensure it doesn’t download with `--no-remote`, but still allows it to work if already cached.

`--cached-only` is for that

Reply:
Oh yeah, you're correct.